website/content/en/docs/setup/quickstart.md

```diff
@@ -79,7 +78,7 @@ The `inputs` option of the `sinks.out` component tells Vector where this sink's
 That's it for our first config. Now let's pipe an event through it:
 
 ```shell
-echo 'Hello world!' | vector --config ./vector.toml
```

Comment:
Curious why the `--config` option not needed here?
I just ran and see it appears to be defaulting to /etc/vector/.

Reply:
The config above ln60 is written locally to `./vector.toml` and then mounted to the `/etc/vector/vector.toml` path due to `-v $(pwd)/:/etc/vector/`

Reply:
I see now. Thanks for clarifying!

Reply:
I don't _super_ love it, but it's functional again. Happy to take other suggestions on cleaning things up.